drivers/media/i2c/imx290.c

```diff
@@ -312,6 +358,11 @@ static const struct imx290_mode imx290_modes_2lanes[] = {
 		.link_freq_index = FREQ_INDEX_1080P,
 		.data = imx290_1080p_settings,
 		.data_size = ARRAY_SIZE(imx290_1080p_settings),
+		.clk_data = {
+			[CLK_37_125] = imx290_37_125mhz_clock_1080p,
+			[CLK_74_250] = imx290_74_250mhz_clock_1080p,
```

Comment:
This should be CLK_74_25, and below. And the _4lanes modes could also do with the same treatment.

Reply:
My compile was still running :-(